spec/spec_helper.rb

```diff
@@ -14,7 +14,7 @@
 
   def assert_correct_indenting(extension='rb', string)
     whitespace = string.scan(/^\s*/).first
-    string = string.split("\n").map { |line| line.gsub /^#{whitespace}/, '' }.join("\n").strip
+    string = string.split("\n").map { |line| line.gsub(/^#{whitespace}/, '') }.join("\n").strip
```

Comment:
Question for a `vim-ruby` maintainer: do you want to keep ruby < 2.3 compatibility for the tests? I can change all the `<<-EOF` to `<<~EOF` if ruby 2.3+ is acceptable for running the tests.